src/service/eol/eol.svc.ts

```diff
@@ -1,40 +1,41 @@
-import { log } from '../../utils/log.util';
-import { daysBetween } from '../../utils/misc';
-import { ScanResult, ScanResultComponent } from '../nes/modules/sbom';
-import { NesApolloClient } from '../nes/nes.client';
-import { createBomFromDir } from './cdx.svc';
-import { Sbom, SbomEntry, SbomMap as SbomModel, ScanOptions } from './eol.types';
-
+import { log } from '../../utils/log.util.ts';
+import { daysBetween } from '../../utils/misc.ts';
+import type { ScanResult, ScanResultComponent } from '../nes/modules/sbom.ts';
+import { NesApolloClient } from '../nes/nes.client.ts';
+import { createBomFromDir } from './cdx.svc.ts';
+import type { Sbom, SbomEntry, SbomMap as SbomModel, ScanOptions } from './eol.types.ts';
 
 const SHOW_OCCURRENCES = (process.env.SHOW_OCCURRENCES || 'false') === 'true';
 const SHOW_OK = (process.env.SHOW_OK || 'false') === 'true';
 
-
 /**
  * Main function to scan directory and collect SBOM data
  */
 export async function scanForEol(directory = process.cwd(), opts: ScanOptions = {}) {
-  const sbom = await createBomFromDir(directory, opts.cdxgen || {})
-  log.info('SBOM generated')
-  const model = await extractComponents(sbom)
-  const scan = await submitScan(model)
+  const sbom = await createBomFromDir(directory, opts.cdxgen || {});
+  if (!sbom) throw new Error('SBOM not generated');
+  log.info('SBOM generated');
```

Comment:
Just as mental note: we can start to switch stuff like this to debug/trace (forget exactly) so that it doesn't show up unless they run with a flag